devices/mini.go

```diff
@@ -0,0 +1,61 @@
+package devices
+
+import (
+	"image"
+
+	streamdeck "github.com/magicmonkey/go-streamdeck"
+)
+
+var (
+	miniName                     string
+	miniButtonWidth              uint
+	miniButtonHeight             uint
+	miniImageReportPayloadLength uint
+	miniImageReportHeaderLength  uint
+	miniImageReportLength        uint
+)
+
+// GetImageHeaderMini returns the USB comms header for a button image for the XL
+func GetImageHeaderMini(bytesRemaining uint, btnIndex uint, pageNumber uint) []byte {
+	thisLength := uint(0)
+	if miniImageReportPayloadLength < bytesRemaining {
+		thisLength = miniImageReportPayloadLength
+	} else {
+		thisLength = bytesRemaining
+	}
+	header := []byte{'\x02', '\x07', byte(btnIndex)}
+	if thisLength == bytesRemaining {
+		header = append(header, '\x01')
+	} else {
+		header = append(header, '\x00')
+	}
+
+	header = append(header, byte(thisLength&0xff))
+	header = append(header, byte(thisLength>>8))
+
+	header = append(header, byte(pageNumber&0xff))
+	header = append(header, byte(pageNumber>>8))
+
+	return header
+}
+
+func init() {
+	miniName = "Streamdeck Mini"
+	miniButtonWidth = 80
+	miniButtonHeight = 80
+	miniImageReportLength = 1024
+	miniImageReportHeaderLength = 16
+	miniImageReportPayloadLength = miniImageReportLength - miniImageReportHeaderLength
```

Comment:
Lines 44-48 are pulled from the Mini's device definition in the python API library for StreamDeck:

https://github.com/abcminiuser/python-elgato-streamdeck/blob/master/src/StreamDeck/Devices/StreamDeckMini.py

I added the `miniImageReportLength` and `miniImageReportHeaderLength` to match the data I found in the python definition. `miniImageReportPayloadLength` is set to the value generated when subtracting the ImageReportHeaderLength from the ImageReportLength